libs/blocks/marketo/marketo.js

```diff
@@ -91,6 +92,35 @@ export const setPreferences = (formData) => {
   Object.entries(formData).forEach(([key, value]) => setPreference(key, value));
 };
 
+const showSuccessSection = (formData, scroll = true) => {
+  try {
+    const show = (el) => {
+      el.classList.remove('hide-block');
+      if (scroll) el.scrollIntoView({ behavior: 'smooth' });
+    };
+    const section = formData[SUCCESS_SECTION].toLowerCase().replaceAll(' ', '-');
+    const success = document.querySelector(`.section.${section}`);
+    if (success) {
+      show(success);
+      return;
+    }
+    // For Marquee use case
+    let count = 0;
+    const interval = setInterval(() => {
+      const el = document.querySelector(`.section.${section}`);
+      if (el) {
+        clearInterval(interval);
+        show(el);
+      }
+      count += 1;
+      if (count > 6) clearInterval(interval);
```

Comment:
nit: might be more readable to set 6 to a var, like `maxIntervals` or something along those lines.